pkg/options/options.go

```diff
@@ -12,26 +12,25 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package maker provides functionality for adding options to components
-package maker
+// Package options provides functionality for adding options to components.
```

Comment:
Please add more context here.  What is an option?  Why?  Examples?

Reply:
+1

Reply:
Added some context